packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -112,6 +112,11 @@ export const esmPatientChartSchema = {
       'An array of concept UUIDs. If an observation has a concept UUID that matches any of the ones in this array, it will be hidden from the observations list in the Encounters summary table.',
     _default: [],
   },
+  viewOnlyVisitLocationField: {
```

Comment:
Pedantic, but perhaps `readOnlyVisitLocationField` might be a better fit here. Additionally, the `ComboBox` component provides a [readOnly](https://react.carbondesignsystem.com/?path=/story/components-combobox--playground&args=disabled:!false;readOnly:!true) boolean prop. Might that be preferable to using `disabled`?

Reply:
```suggestion
  disableChangingVisitLocation: {
```